packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -732,8 +731,10 @@ Future<TimeOfDay> showTimePicker({
   TimeOfDay initialTime
 }) async {
   assert(initialTime != null);
-  return await showDialog(
+  TimeOfDay picked = await showDialog(
     context: context,
     child: new _TimePickerDialog(initialTime: initialTime)
-  ) ?? initialTime;
```

Comment:
Same comments here.  We can return the result of the `await` directly without the `??` or the local variable.